cmd/check_container.go

```diff
@@ -157,106 +155,43 @@ var checkContainerCmd = &cobra.Command{
 			log.Info("preparing results that will be submitted to Red Hat")
 			log.Tracef("CertProject: %+v", certProject) // TODO() This depends on Pyxis calls made outside of this block.
 
-			// The engine writes the certified image config to disk in a Pyxis-specific format.
-			// Include that data in our submission.
-			certImageFilepath := path.Join(artifacts.Path(), certification.DefaultCertImageFilename)
-			log.Tracef("Loading certImage data from path %s", certImageFilepath)
-			certImageBytes, err := os.ReadFile(certImageFilepath)
+			submission, err := pyxis.NewCertificationInput(
+				certProject, // Note: certProject depends on Pyxis calls made outside of this block.
```

Comment:
If #558 is merged this note can go away correct?

Reply:
Yes. In fact, I think I took it out in #558